jaeger-core/src/test/java/com/uber/jaeger/reporters/RemoteReporterTest.java

```diff
@@ -62,7 +68,7 @@ public void setUp() throws Exception {
   public void testRemoteReporterReport() throws Exception {
     Span span = (Span) tracer.buildSpan("raza").start();
     reporter.report(span);
-    Thread.sleep(5);
+    Thread.sleep(50);
```

Comment:
I increased it because it was failing every time on my laptop. There's already a PR to make it right: https://github.com/uber/jaeger-client-java/pull/177